openbts/components.py

```diff
@@ -541,6 +541,14 @@ def get_gprs_usage(self, target_imsi=None):
       except ValueError:
         # Casting to int failed.
         continue
+      # See if we already have an entry for the same IMSI -- we sometimes see
+      # duplicates.  If we do have an entry already, take the greater value of
```

Comment:
if all the traffic comes from the same IMSI, why not just sum the entries together?